v7/_posts/release notes/2018-01-11-ODL-7.4.0.beta3.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: "ODataLib 7.4.0.beta3"
+description: "ODataLib 7.4.0.beta3 release notes"
+category: "4. Release Notes"
+---
+
+## Changes in ODataLib 7.4.0.beta3 Release ##
+
+## Fixed Bugs ##
+
+[[Issue #1022](https://github.com/OData/odata.net/issues/1022)] Calculate correct context URI with Operation path segment.
+
+[[Issue #1193](https://github.com/OData/WebApi/issues/1193)] Add the extension function back to derived class.
+
+[[Issue #1028](https://github.com/OData/odata.net/issues/1028)] Add the enum member expression into validation and don't return the error message.
+
+This release delivers OData core libraries including ODataLib, EdmLib and Spatial. OData Client for .NET is not published in this release.
```

Comment:
We actually should publish a client with this release.

Reply:
+1

Maybe we can also update the library names to something like "This release delivers OData libraries, including Client, Core, Edm, and Spatial."